### DISCUSSION TITLE
Failed to run locally.

### DISCUSSION THREAD

**Discussion by aniljangirdev** [CrawlerCode/redmine-time-tracking#19] [Q&A] [answered]
Hello, I want to start contribute on this plugin.

any one can tell me how to run this project locally?

Thanks

**Comment by CrawlerCode** [comment, marked as answer]
First you need to install [Node.js](https://nodejs.org/en) and [pnpm](https://pnpm.io/).

1. Install the dependencies with the following command:
`pnpm install`

2. Start project in dev mode:
`pnpm run dev`
_The extension will be created and you can find all the files of the chrome extension in the `dist` folder._

3. Add the extention to your chrome browser:
- Open your chrome browser and navigate to your extensions
- Enable `Developer mode`
- Click `Load unpacked` and select the `dist` folder

![image](https://github.com/CrawlerCode/redmine-time-tracking/assets/41094392/a64e8dac-ed7d-4fbe-b728-49a74c76edca)

**Reply by aniljangirdev** [reply]
thank you very much for your help.
i appreciate it @CrawlerCode
